src/brain_atlas/util/hail_batch/13_precalc_for_hail/hail_diffexp_runner.py

```diff
@@ -0,0 +1,195 @@
+import pickle
+
+import numpy as np
+import pandas as pd
+
+from tqdm import tqdm
+
+import dask
+import dask.array as da
+import dask.distributed
+
+# Maybe faster, equivalent API
+import _pickle as cPickle
```

Comment:
FYI in python 3 the regular pickle module uses the C implementation already. This difference was relevant in python 2